util/Xamarin.AndroidBinderator/Xamarin.AndroidBinderator/Extensions.cs

```diff
@@ -123,17 +123,24 @@ public ProjectResolver (CachedMavenRepository repository)
 
 	public Project Resolve (Artifact artifact)
 	{
```

Comment:
This file is not the correct place the error occurs, the build still just says this during the `binderate` cake target:
```
Dependency errors : 1
1
	System.Net.Http.HttpRequestException: Response status code does not indicate success: 404 (Not Found).
   at System.Net.Http.HttpResponseMessage.EnsureSuccessStatusCode()
   at System.Net.Http.HttpClient.GetStreamAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
```

Undo the changes in this file, and log in the correct location.

If you can't figure out the appropriate place to add logging, tell me plausible locations we should look.